src/wallet/test/coinselector_tests.cpp

```diff
@@ -1183,6 +1183,14 @@ BOOST_AUTO_TEST_CASE(coin_grinder_tests)
             return available_coins;
         });
         BOOST_CHECK(res);
```

Comment:
nit: Maybe this `BOOST_CHECK` is not needed anymore since you're checking the result itself.

Reply:
Good idea, thanks.   I've removed the implicit check.